### ISSUE TITLE
Create Priodiversity data management plan (DMP, dDMP) tool

### ISSUE THREAD

**Issue by wkmor1** [luomus/meta#33]
Priodiversity DMP tool is an application used by Priodiversity organizations to save and edit their data management plans. maDMPs are machine actionable, meaning that they're available in JSON format, constructed with controlled vocabulary (enums/options) instead of text fields. The system includes a front-end for filling/editing DMP forms, and an API for fetching the JSON maDMPs.

Deliverable D2.2
Dynamic Data Management Planning (dDMP) tool for ReBAPs’ related data.
Lead: ELY
M96

Blocks https://github.com/luomus/meta/issues/211

- [x] depoying to openshift
- [x] Deliverable report
- [x] visual design
- [x] logos
- [x] tietosuojaseloste (link to  https://info.laji.fi/etusivu/lajitietokeskus/tietosuojaseloste/ )
- [x] saavutettavuusseloste (Robert)
- [x] add to plausible

**Comment by wkmor1** [comment]
Can we have a more informative title and description for this Epic?

**Comment by rpulkka** [comment]
Waiting for the OSTrails schema.

**Comment by esko-piirainen** [comment]
@rpulkka Creates tasks for @Blodir and go over them; few details to be implemented in the coming spring 4.2. ->

**Comment by esko-piirainen** [comment]
Authentication / authorization is done
Technical development is now blocked
Next up: Robert compiling uniform data model for Anne to comment

**Comment by esko-piirainen** [comment]
Anne has commented data model and it has been now moved to Syke, MH etc for comments

**Comment by esko-piirainen** [comment]
Schema design is done. Create table statements done by Robert.
Next up Pyry will work on backend.

**Comment by mikkohei13** [comment]
API done, PM will work on frontend and ML on containerization.

Deadline has been postponed to end of September.

**Comment by Blodir** [comment]
Robert will continue talks with Syke about future tasks

Markus will deploy prod

**Comment by rpulkka** [comment]
Data connection fixes are done. UI fixes to be done (EU / Natura logo at least).

**Comment by rpulkka** [comment]
Tietosuojaseloste (link to info.laji.fi), saavutettavuusseloste (needs testing etc.), Plausible.

**Comment by aorin** [comment]
Footer is done, tietosuojaseloste, saavutettavuusseloste and Plausible waiting to be done

**Comment by Blodir** [comment]
Still waiting on accessibility document and plausible. Also there's an issue with corrupt data on dev...

**Comment by MLumme** [comment]
Accessibility work ongoing.

**Comment by rpulkka** [comment]
No need to fix all accessibility issues, just need to write the report.
